samples/Extensions/Http/HttpFunction.cs

```diff
@@ -21,10 +21,10 @@ public static class HttpFunction
             //</docsnippet_logging>
 
             var response = req.CreateResponse(HttpStatusCode.OK);
-            response.Headers.Add("Date", "Mon, 18 Jul 2016 16:06:00 GMT");
+           
             response.Headers.Add("Content-Type", "text/plain; charset=utf-8");
 
-            response.WriteString("Welcome to .NET 5!!");
+            response.WriteString("Welcome to .NET isolated worker !!");
```

Comment:
isolated or Isolated ?

Reply:
isolated

Reply:
usually, we do capitalize  the first letters for the product name but we don't have an Isolated Worker ( IW ) for us so in this scenario : isolated is fine